packages/widgets/src/components/NotificationMenu/helpers.ts

```diff
@@ -0,0 +1,97 @@
+import { ButtonGroupItem } from '@ssa-ui-kit/core';
+import { NotificationCardProps } from '@components/NotificationCard';
+
+export interface DataProps {
+  notificationItems: NotificationCardProps[];
+  groupButtonItems: ButtonGroupItem[];
+}
+
+export const notificationData: DataProps = {
+  groupButtonItems: [
+    { id: 1, text: 'All', isDisabled: false },
+    { id: 2, text: 'Unread', isDisabled: false },
+  ],
+  notificationItems: [
+    {
+      title: 'lorem',
+      text: 'With our newest listing, we’re welcoming Wrapped Bitcoin to innovation Zone! You can now deposit',
+      badgeColor: 'blueLight',
```

Comment:
Let's combine these two properties (`badgeColor` and `iconName`) together into the `type` field.

![image](https://github.com/ssagroup/ui-kit/assets/82376677/4c42f3e7-5b4a-47ae-838d-60de71f89cde)

Reply:
Done